AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/ReconcileAndLocalSave/AWSModelReconciliationQueue.swift

```diff
@@ -93,11 +93,7 @@ final class AWSModelReconciliationQueue: ModelReconciliationQueue {
             .sink(receiveCompletion: { [weak self] completion in
                 self?.receiveCompletion(completion)
                 }, receiveValue: { [weak self] receiveValue in
```

Comment:
1. nit: arg name is probably better as `value` 
1. For future: I poked at this a bit today, and I don't think we need to `weak self` inside the closure--as long as we're properly disposing of the cancellable, and as long as our owner is properly disposing of us, we don't get a retain cycle from this closure. When I did this originally I didn't investigate it first, and it looks like I was overabundantly cautious (or as we call it, "paranoid")

Reply:
thanks for the clarification, i'll put this on the todo list.